staging/src/k8s.io/apiserver/pkg/server/options/serving.go

```diff
@@ -83,6 +86,7 @@ func NewSecureServingOptions() *SecureServingOptions {
 			PairName:      "apiserver",
 			CertDirectory: "apiserver.local.config/certificates",
 		},
+		HTTP2MaxStreamsPerConnection: 1000,
```

Comment:
Of course this affects memory, CPU, and network load.  I agree it would be better for each program to have its own default.  It would be even better if a server could tailor this limit to each individual client.  What would you suggest as the API for either of these?

Reply:
Yes, I will revise to do that.